test/format/wts.c

```diff
@@ -176,7 +176,7 @@ static WT_EVENT_HANDLER event_handler = {
  *     Open a connection to a WiredTiger database.
  */
 void
-wts_open(const char *home, bool set_api, WT_CONNECTION **connp)
+wts_open(const char *home, bool set_api, WT_CONNECTION **connp, bool no_verify)
```

Comment:
I'd prefer something like `allow_verify`. Passing `false` to this is a double negative.